documentation/references.md

```diff
@@ -15,13 +18,22 @@ If you applied AMICI in your work and your publication is missing, please let us
 <div id="ref-StaedterSch2021">
 <p>Städter, Philipp, Yannik Schälte, Leonard Schmiester, Jan Hasenauer, and Paul L. Stapor. 2021. “Benchmarking of Numerical Integration Methods for Ode Models of Biological Systems.” <em>Scientific Reports</em> 11 (1): 2696. <a href="https://doi.org/10.1038/s41598-021-82196-2">https://doi.org/10.1038/s41598-021-82196-2</a>.</p>
 </div>
+<div id="ref-StenPod2021">
+<p>Sten, Sebastian, Henrik Podéus, Nicolas Sundqvist, Fredrik Elinder, Maria Engström, and Gunnar Cedersund. 2021. “A Multi-Data Based Quantitative Model for the Neurovascular Coupling in the Brain.” <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2021.03.25.437053">https://doi.org/10.1101/2021.03.25.437053</a>.</p>
```

Comment:
Is it intentional that many publications appear to have the first author as `[Last Name], [First Name]` then subsequent authors as `[First Name] [Last Name]`?

Reply:
Not my favorite, but that's what pandoc-citeproc does :)